telco-core/configuration/reference-crs-kube-compare/metadata.yaml

```diff
@@ -40,13 +40,17 @@ parts:
           - path: required/networking/metallb/bfd-profile.yaml
           - path: required/networking/metallb/bgp-advr.yaml
           - path: required/networking/metallb/bgp-peer.yaml
-          - path: required/networking/metallb/community.yaml
           - path: required/networking/metallb/metallb.yaml
           - path: required/networking/metallb/metallbNS.yaml
           - path: required/networking/metallb/metallbOperGroup.yaml
             config:
               ignore-unspecified-fields: true
           - path: required/networking/metallb/metallbSubscription.yaml
+      - name: optional-networking-metallb
```

Comment:
Wondering if a better place for this would be under the `- name: optional-networking` section.